singular.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * The template for displaying all pages, single posts and attachments
+ *
+ * This is a new template file that WordPress introduced in
+ * version 4.2. Note that it uses conditional logic to display
```

Comment:
Yes, that's correct that's why it won't give us a fatal. We all agree a new core theme shouldn't be facilitating use of older core versions, and making the theme work as expected in 4.2 makes index.php a lot bigger and that's going to be a cruft in my opinion. We can debate in a separate thread if we want but for now, we should just add singular.php and remove page.php and single.php, and leave index.php as it is.